plugins/janus_audiobridge.c

```diff
@@ -1138,6 +1141,8 @@ typedef struct janus_audiobridge_room {
 	gchar *record_file;			/* Path of the recording file */
 	FILE *recording;			/* File to record the room into */
 	gint64 record_lastupdate;	/* Time when we last updated the wav header */
+	gboolean wav_header_added;  /* if wav header is added in recording file */
+	gint64 rec_start_time;		/* time when recording started for generating file name */
```

Comment:
Code style: comments should be separated by the code via tabs, not spaces. Besides, all comments should start with a capitalized letter.

Reply:
done